contracts/schemes/GenericSchemeMultiCall.sol

```diff
@@ -153,6 +153,9 @@ contract GenericSchemeMultiCall is VotingMachineCallbacks, ProposalExecuteInterf
                 (success, genericCallReturnValue) =
                 controller.genericCall(proposal.contractsToCall[i], callData, avatar, proposal.values[i]);
             }
+
+            /* Whole transaction will be reverted if at least one call fails*/
+            require(success, "Proposal call failed");
```

Comment:
please remove the automatic call to `execute`  at https://github.com/daostack/arc/blob/b80fe92ff7dd9eb069c7770431c5dfb041d3e9e1/contracts/schemes/GenericSchemeMultiCall.sol#L116 .
this is not good due to :
1. last voter(executer) does not need to pay for whole execution gas.
2. if execution is reverted in one of the calls.. so the dao cannot decide (for the case of absolute majority)..
 - there suppose to be a differentiation between the dao decision and execution.